contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -336,6 +342,7 @@ GetBulkLoadRequest(StringInfo message)
 			{
 				uint16 plp;
 				CheckMessageHasEnoughBytesToRead(&message, BINARY_COLUMNMETADATA_LEN);
+				Assert(sizeof(uint16)<=sizeof(plp));
```

Comment:
Redundant, always true
We have declared uint16 plp; and then checking (sizeof(uint16)<=sizeof(plp), why?